### PR TITLE
Adding snippets

### PR DESCRIPTION
**Pull request by deanMaker**
Adding commonly used snippets to project.

### CODE REVIEW

**Comment by amykyta3** [comment]
Thanks! I'll experiment with this in a few days once I have some time. Generally looks good but I may opt to simplify some of the snippets for more general usage.

File: snippets/systemrdl.json [hunk 1]
```diff
@@ -0,0 +1,124 @@
+{
+    "Addrmap": {
+        "prefix": ["addrmap"],
+        "body": ["addrmap ${1:blkname} {", 
+            "\tname = \"$2\";",
+            "\tdesc = \"$3\";",
+            "\tdefault regwidth = 32;",
+            "\tdefault sw = rw;",
+            "\tdefault hw = r;",
+            "\trsvdset = 0;",
```
Comment (amykyta3) [inline snippets/systemrdl.json]: Would prefer the addrmap snippet to not include as many properties at the outset as they are not as universally used.

Also, the addrmap snippet appears to be missing the closing curly brace `}`

Comment (deanMaker) [reply snippets/systemrdl.json]: Good suggestions.  Fixed

Comment (deanMaker) [reply snippets/systemrdl.json]: Alex,
Any feedback?

**Comment by amykyta3** [comment]
Merged and deployed to v1.2.0 of the extension.
Thanks!

**Pull request merged**